specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7018,17 +7018,17 @@
       "properties": {
         "lineEnding": {
           "$ref": "#/definitions/OcrSkillLineEnding",
-          "description": "A value indicating which line ending to use. Default is space."
+          "description": "A value indicating which line ending to use. Default is \"space\"."
         },
         "defaultLanguageCode": {
           "$ref": "#/definitions/OcrSkillLanguage",
-          "description": "A value indicating which language code to use. Default is en."
+          "description": "A value indicating which language code to use. Default is \"en\"."
         },
         "detectOrientation": {
           "x-ms-client-name": "ShouldDetectOrientation",
           "type": "boolean",
           "default": false,
-          "description": "A value indicating to turn orientation detection on or not. Default is false."
+          "description": "A value indicating to turn orientation detection on or not. Default is \"false\"."
```

Comment:
Nit: not sure I'd quote false since this is a boolean property, but since it's used as a value, might be considered consistent.